### ISSUE TITLE
Add toggle to beautify response

### ISSUE THREAD

**Issue by codymullins** [spacetimedotnet/spacetime#11]
We added the ability to beautify responses in #10.

Implement the toggle setting using the "Paint" icon on the Response toolbar.

- [ ] Add toggle (either globally or per request)
- [ ] Save status of toggle
- [ ] Add unit tests for beautify + toggle
- [ ] This toggle should take effect during the request

![image](https://user-images.githubusercontent.com/1738479/166843197-cd7cb2cc-ef19-44fe-a95e-45f322910f88.png)

**Comment by michel-tr** [comment]
Let's see what I can do. @codymullins Do you have a chat channel (Discord, Slack, etc) to communicate and discuss stuff?

**Comment by codymullins** [comment]
Good idea! I've created a Discord server: https://discord.gg/PeBbYy6WKq
